cumulus/polkadot-parachain/src/chain_spec/people.rs

```diff
@@ -23,6 +23,10 @@ use std::str::FromStr;
 /// Collects all supported People configurations.
 #[derive(Debug, PartialEq)]
 pub enum PeopleRuntimeType {
+	Kusama,
+	KusamaLocal,
```

Comment:
We should not include the local variants. If people need them, they need to create the chain specs themselves. Most of the time you want for the local variants any way the latest code.

Reply:
Don't the benchmark scripts use local?

Reply:
Well, I just remember @serban300's [PR](https://github.com/paritytech/polkadot-sdk/pull/3443) where he had to add this local variants to be able run fellows zombienet tests for BridgeHubKusama/BridgeHubPolkadot.

We don't provide chain-specs for local variants here as I explained bellow: https://github.com/paritytech/polkadot-sdk/pull/3961#discussion_r1549542943.

These local variants are needed for parsing `id` string from chain-spec and used for managing what kind of node to run here: https://github.com/paritytech/polkadot-sdk/pull/3961/files#diff-517780020940588a81c29ec543f6c5dff196c985641a28826c955ac35d1aa02bR761-R764


I think, `polkadot-parachain-bin` should be maybe chain-agnostic and we should provide just chain-spec json file. So we should remove all these chain-specific stuff from `polkadot-parachain` and extract that to new crate `chain-spec-generator` for testnets as we have for [fellows](https://github.com/polkadot-fellows/runtimes/tree/main/chain-spec-generator).
And also, for controlling what setup to run `start_generic_aura_node` vs `start_generic_aura_lookahead_node` vs `start_asset_hub_node`  vs `start_asset_hub_lookahead_node`, we could add new command parameter (or maybe attribute to chain-spec?).

Reply:
We can improve and refactor whatever way, but let's continue here https://github.com/paritytech/polkadot-sdk/issues/3944 or is there any other issue?

Reply:
> Don't the benchmark scripts use local?

I think you need just chain-spec for benchmarks, e.g for fellows benchmarks, I just needed `chain-spec-generator` to generate a local chain spec for `coretime-kusama-local`, and then use `polkadot` binary:
```
./polkadot benchmark pallet \
    --chain=./$CHAIN-chain-spec.json
```

I didn't try to run benchmarks with `polkadot-parachain`, anyway [`omni-bencher`](https://github.com/paritytech/polkadot-sdk/pull/3512) is coming, so we won't even need chain-spec for benchmarking